<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>3D试眼镜DEMO</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="MainGlwight" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>5</y>
      <width>681</width>
      <height>511</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>91</width>
      <height>506</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTextBrowser" name="textBrowser"/>
     </item>
     <item>
      <widget class="QPushButton" name="shoot_button">
       <property name="text">
        <string>拍摄</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="reshoot_button">
       <property name="text">
        <string>重新拍摄</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="confirm_image">
       <property name="text">
        <string>确认图片</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>90</width>
      <height>508</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MainGlwight</class>
   <extends>QOpenGLWidget</extends>
   <header>MainGlwight.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>shoot()</slot>
 </slots>
</ui>
